### ISSUE TITLE
Doesn't inline JSX if less than 80 chars (and contains newline)

### ISSUE THREAD

**Issue by knowbody** [prettier/prettier#2001]
I'm having an issue where if my code looks like:

```js
<P>
  Hello world</P>
```

currently prettier would format it like:

```js
<P>
  Hello world
</P>
```

if however I have something like:

```js
<P>Hello world</P>
```

it will keep it this way, and this is the behaviour that I would expect initially. So I'd like:

```js
<P>
  Hello world</P>
```

to be formatted to:

```js
<P>Hello world</P>
```

**Comment by karl** [comment]
Prettier currently maintains all newline characters in JSX, which gives the behaviour you are seeing above. If you want Prettier to inline JSX you'll need to remove those line breaks manually.

We are investigating having Prettier discard existing line breaks when formatting, you can follow the progress in this PR: https://github.com/prettier/prettier/pull/1831

**Comment by vjeux** [comment]
This is going to be fixed by #2001, closing this one.
